framework/test/functional/mysql.test.js

```diff
@@ -196,6 +196,12 @@ describe('Test MySQL', () => {
 			expect(result).toBeInstanceOf(Array);
 			expect(result.length).toBe(2);
 		});
+
+		it('Distinct query', async () => {
+			await testTable.upsert([emptyBlock, { ...nonEmptyBlock, id: '5683515590859200000' }]);
```

Comment:
Please ensure that `id` is not the primary key for the `testTable`. It would be better to reuse some other property. Let us discuss on how to update the test offline.